src/components/_navbar.scss

```diff
@@ -323,12 +323,13 @@
   &[data-toggle="collapse"] {
     
```

Comment:
maybe it's just me, but I wouldn't use this kind of attributes to add styles. other framework implementations could not need these

Reply:
This is specifically a Bootstrap theme and that is one of the attributes Bootstrap uses to implement some of its functionality, so I see no problem using it.